src/Control/FunFlow/External/Executor.hs

```diff
@@ -121,10 +122,12 @@ executeLoop _ cfg sroot = do
           afterFailure t i = Failed (ExecutionInfo executor t) i
 
       CS.withStore sroot $ \store -> forever $ do
+        $(logTM) InfoS "Awaiting task from coordinator."
         mtask <- popTask hook executor
```

Comment:
Looks like a blocking `waitForNewTask` might be good at some point.

Reply:
Concur.